### DISCUSSION TITLE
Welcome to odf-report Discussions!

### DISCUSSION THREAD

**Discussion by sandrods** [sandrods/odf-report#111] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jctampa** [comment]
Image naming. It is not clear on how to 'NAME' an image. Is this supposed to be the filename? No extension? Caption? 
How would one NAME an image in libreoffice or word?

"You must put a mock image in your .odt template and give it a name. That name will be used to replace the mock image for the actual image."

Tks

**Reply by Speedway1** [reply]
Is anyone still here?
